docs/getting-started.md

```diff
@@ -80,14 +90,48 @@ public class MainActivity extends ReactActivity {
 }
 ```
 
+##### Running jetifier
+
+> **Important:** This step is only necessary if you're on React Native 0.60 or greater.
+
+React Native 0.60 migrated from Support Library to AndroidX. React Native Gesture Handler is not yet compatible with AndroidX. Until then, please use [`jetifier`](https://www.npmjs.com/package/jetifier) to make it work with newest React Native:
+
+```sh
+yarn add jetifier
+```
+
+or with npm:
+
+```sh
+npm install --save-dev jetifier
+```
+
+We recommend using "postinstall" phase to run it. You can add it in your `package.json` file:
```

Comment:
I prefer to add this step **before** running `yarn add jetifier` so it's automatically run. Or we need to include extra step `yarn jetify` after this step so the dep is jetified.

Reply:
Oh yea, good call